### ISSUE TITLE
Feat: Ensure input in any long field / wysiwyg component handles localStorage to avoid loss of data

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#4395]
## Feature description and goals

Currently, localStorage is handled per usage of TinyMCE. Any new integration of TinyMCE must manually handle its own localStorage to save input from being lost if a crash / tab close happens.

Something should be done to handle ensure localStorage is automatically handled when integrating tinyMCE in a new place, as well as for any long input.

**Comment by Haskaris** [comment]
Here is a plugin we can use with TinyMCE: https://www.tiny.cloud/docs/plugins/opensource/autosave/

**Comment by lebouquetin** [comment]
I dont think we should use TinyMCE autosave. We definitly need to talk about putting tinyMCE everywhere.
